airflow/configuration.py

```diff
@@ -456,7 +456,7 @@ def getimport(self, section, key, **kwargs):
         """
         Reads options, imports the full qualified name, and returns the object.
 
-        In case of failure, it throws an exception a clear message with the key aad the section names
+        In case of failure, it throws an exception a clear message with the key and the section names
```

Comment:
```suggestion
        In case of failure, it throws an exception with the key and section names
```

Might as well fix the grammar while we are at it?